src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/mobile/create_window_request_test.cc

```diff
@@ -293,6 +293,78 @@ TEST_F(
   command->Run();
 }
 
+TEST_F(
+    CreateWindowRequestTest,
+    Run_DuplicateUpdatesFromMainWindowWithActiveWebViewTemplate_ExpectRejectedCreateWindow) {
+  EXPECT_CALL(
+      mock_rpc_service_,
+      ManageHMICommand(HMIResultCodeIs(hmi_apis::FunctionID::UI_CreateWindow),
+                       Command::CommandSource::SOURCE_SDL_TO_HMI))
+      .Times(0);
+
+  const auto mobile_result_code = mobile_apis::Result::REJECTED;
+  EXPECT_CALL(
+      mock_rpc_service_,
+      ManageMobileCommand(CheckMessageToMobile(mobile_result_code, false),
+                          Command::CommandSource::SOURCE_SDL))
+      .WillOnce(Return(true));
+
+  const auto default_window_id = mobile_apis::PredefinedWindows::DEFAULT_WINDOW;
+  ON_CALL(*mock_app_, WindowIdExists(default_window_id))
+      .WillByDefault(Return(true));
+  ON_CALL(*mock_app_, name()).WillByDefault(ReturnRef(kAppName));
```

Comment:
@AByzhynar
`  ON_CALL(*mock_app_, WindowIdExists(kTestWindowId)).WillByDefault(Return(false));`
Is it needed here?

Reply:
@mvorobio Thank you. Added in c0ffac5e7d5b42371817ee58acb097990b1a7bed